### ISSUE TITLE
Name of recipe is being announced by Voiceover but its not visible for visual users.

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#5814]
GitHubTags:
#.NET MAUI; #A11ySev4; #A11yMAS; #A11yUsable; #A11yTCS; #Benchmark; #iOS; #iPad; #GH_.NETMAUI_IOS_Mar2022; #AINotInScope;

### Description

After performing search operation we are getting search results in search results name of recipe is being announced by Voiceover but its not visible for visual users.

### Steps to Reproduce

1.Install and open ".NET MAUI" app.
2.Swipe till "search box".
3.Enter Recipe name in search box and search.
4.Verify Recipe Name is Visible or Not.

https://user-images.githubusercontent.com/100186395/161682772-c2535c1b-4b26-4bcd-abec-821dd02f1e42.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 15.4(iPad)

### Relevant log output

```shell
Name of recipe is being announced by Voiceover it should be visible for visual users.
```

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI" app. issue is fixed so closing the bug
IOS Version: iPad(15.5)
added attachment for references

https://user-images.githubusercontent.com/100186395/177999814-95a16610-3119-4e27-8923-4830f14eb710.mp4
